Mathlib/Util/LeanSearchExamples.lean

```diff
@@ -0,0 +1,43 @@
+/-
+Copyright (c) 2024 Siddhartha Gadgil. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Siddhartha Gadgil
+-/
```

Comment:
Please move this file to `tests`.

Reply:
Done. Moved to `test/LeanSearchExamples.lean`